README.md

```diff
@@ -36,11 +36,18 @@ Every member of the product team is expected to work hard, be considerate of the
 
-Furthermore, any behavior or language which is unwelcoming—whether or not it rises to the level of harassment—is also strongly discouraged. Much exclusionary behavior takes the form of [microaggressions](http://www.vox.com/2015/2/16/8031073/what-are-microaggressions)—subtle put-downs which may be unconsciously delivered. Regardless of intent, microaggressions can have a significant negative impact on victims and have no place on our team.
+Any behavior or language that is unwelcoming is strongly discouraged. Such exclusionary behavior can take many forms, including [microaggressions](http://www.vox.com/2015/2/16/8031073/what-are-microaggressions)—subtle put-downs, whether deliberate or unconscious. Regardless of intent, microaggressions can have a significant negative impact on victims and have no place on our team.
 
-There are a host of behaviors and language common on tech teams which are worth noting as specifically unwelcome: Avoid “_well, actuallys_”—pedantic corrections that are often insulting and unproductive; make an effort not to interrupt your colleagues while they are speaking; never respond with surprise when someone asks for help; and take care neither to patronize your colleagues nor assume complete knowledge of a topic. This last point is especially important when talking about technical topics: Many women and people of color in the tech industry have many tales of being either [mansplained](https://www.guernicamag.com/daily/rebecca-solnit-men-explain-things-to-me/) about a field in which they are experts, or else excluded from learning opportunities because a colleague wouldn’t make an effort to answer questions—don’t be that person. Remember that your colleagues may have expertise you are unaware of, and listen at least as much as you speak.
+There are a host of behaviors and language common on tech teams which are worth noting as specifically unwelcome:
+
+- avoid “_well, actuallys_”—pedantic corrections that are often insulting and unproductive;
+- make an effort not to interrupt your colleagues while they are speaking;
+- never respond with surprise when someone asks for help;
+- and take care neither to patronize your colleagues nor assume complete knowledge of a topic.
+
+This last point is especially important when talking about technical topics: Women and people of color in the tech industry have many tales of being either [mansplained](https://www.guernicamag.com/daily/rebecca-solnit-men-explain-things-to-me/) about a field in which they are experts, or else excluded from learning opportunities because a colleague wouldn’t make an effort to answer questions. Don’t be that person. Remember that your colleagues may have expertise you are unaware of, and listen at least as much as you speak.
```

Comment:
```This last point is especially important when talking about technical topics: Engineers have many tales of being patronized about a field in which they are experts, or else excluded from learning opportunities because a colleague wouldn’t make an effort to answer questions. Don’t be that person. Remember that your colleagues may have expertise you are unaware of, and listen at least as much as you speak.```

how about that? i just think its more important to convey problems with the message delivery than the messenger